### PR TITLE
User story 32

### LINKED ISSUE #33: User Story 32
Image

As an admin,
When I visit the admin Invoices index (/admin/invoices)
Then I see a list of all Invoice ids in the system
Each id links to the admin invoice show page

### PR DESCRIPTION
**Pull request by westonio**
# Describe the changes below:
- Adds tests, controller action, and view for Admin Invoices #index page
- Adds partial for admin views, and css styling
- NOTE: I accidentally submitted the controller file and tests directly to main 🤦‍♂️ 😢 sorry!
# Relevant user story:
```
32. Admin Invoices Index Page
As an admin,
When I visit the admin Invoices index (/admin/invoices)
Then I see a list of all Invoice ids in the system
Each id links to the admin invoice show page
```

# Before submitting, check the following:
- [x] Entire test suite is passing
- [100%] SimpleCov test percentage

### CODE REVIEW

File: app/views/shared/_admin_topper.html.erb
Comment (MWoodshop) [inline app/views/shared/_admin_topper.html.erb:1]: Nice job - I think I can leverage this and what I wrote before to make a partial for the non-admin views as well.

**Review by MWoodshop — APPROVED** [review]
Looks good to me (especially that CSS :))

File: app/views/shared/_admin_topper.html.erb
Comment (westonio) [reply app/views/shared/_admin_topper.html.erb:1]: Thank you! We can always sync up our css and views as a refactor effort later. I like the way you formatted and labeled a lot of your stuff too.

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -1,15 +1,83 @@
...
+  margin-bottom: 120px;
+}
+
+.current-menu-item a {
+  background: #D8DAD3 ;
+  padding: 10px;
+  border-radius: 10px 100px / 120px;
+  color: black;
+}
+
```
Comment (dani-wilson) [inline app/assets/stylesheets/application.css:71]: nice CSS! Woohoo!

File: app/views/admin/invoices/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<%= render partial: "shared/admin_topper", locals: { invoices: true  } %>
+
+<h3 class="full-length-header"> Invoices </h3>
+
+<div class="admin-invoices">
```
Comment (dani-wilson) [inline app/views/admin/invoices/index.html.erb:5]: Nice use of partials :))

**Review by dani-wilson — APPROVED** [review]
Looks great! Awesome work

**Pull request merged**